config/routes.rb

```diff
@@ -8,7 +8,7 @@
 
   post '/locks/from_pool(.:format)' => 'locks#lock_from_pool', as: "create_from_pool"
 
+  match '/pools/disable(.:format)' => 'pools#disable', via: [:put, :patch], as: "disable_pool"
   resources :pools
```

Comment:
@akostadinov Yeah, exactly. Just was confused that it works for `locks` fine, but in `locks` you changing the same `update` method..